crates/medmodels-core/src/medrecord/group_mapping.rs

```diff
@@ -113,6 +182,21 @@ impl GroupMapping {
         }
     }
 
+    pub fn remove_edge(&mut self, edge_index: &EdgeIndex) {
```

Comment:
I do not know if the linting of this is wrong, but it appears as it is all strings, correct me if I am wrong
![image](https://github.com/limebit/medmodels/assets/60970451/e46e4346-f602-4d30-8567-f8cd22110cfa)

Reply:
Thats just the GitHub parser doing funny things :)